aten/src/ATen/native/cuda/DistributionNormal.cu

```diff
@@ -55,6 +56,13 @@ void normal_kernel_cuda(TensorIterator& iter, double mean_, double std_, Generat
 
 Tensor& normal_cuda_(Tensor& self, double mean, double std, Generator* gen) {
   TORCH_CHECK(std > 0.0, "normal_ expects std > 0.0, but found std=", std);
+  if(self.is_complex()) {
+    // note: float_tensor lives only as long as the self tensor lives
+    auto float_tensor = at::native::view_complex_as_float(self);
+    // variance for normal distribution of the real and imaginary values
+    // is half of the input variance
+    return normal_cuda_(float_tensor, mean, std/(std::sqrt(2)), gen);
```

Comment:
I suspect that problem is that you return float tensor instead of complex in inplace op [randn](https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/native/TensorFactories.cpp#L584), try this:
```suggestion
    normal_cuda_(float_tensor, mean, std/(std::sqrt(2)), gen);
    return self;
```

Reply:
thanks!! you're right. that worked